elements/campaign-display/campaign-display.scss

```diff
@@ -1,8 +1,24 @@
 campaign-display {
-  display: block;
-
+  color: rgba(112, 112, 112, 1);
+  display: inline-block;
+  font-family: 'Helvetica', Arial, sans-serif;
+  font-size: 10px;
+  text-transform: uppercase;
+  .inner a {
```

Comment:
[confirmed](http://www.clickhole.com/search?q=sponsored+by), CH is picking up `.inner` twice:

<img width="485" alt="screen shot 2017-02-27 at 11 05 41 am" src="https://cloud.githubusercontent.com/assets/3171252/23371141/bb02f8dc-fcdc-11e6-988d-dc994b9dbc2f.png">